config/locales/user_authorization_confirmation/en.yml

```diff
@@ -1,7 +1,7 @@
 ---
 en:
   user_authorization_confirmation:
-    continue: Sign in
+    continue: Continue
```

Comment:
Should we just use an existing/common continue button?

Reply:
We could, though we do run the risk of changing that having side effects on this screen where it is presented a little differently than it is elsewhere.

Reply:
Fwiw if we use common Continue button, we should make sure it's still center-aligned (usually it's left aligned, but here it's centered).